tests/test_configs/autoguard/config.yml

```diff
@@ -4,67 +4,233 @@ models:
+            input:
+                guardrails_config:
+                    {
+                      "pii_fast": {
+                          "enabled_types": [
+                              "[BANK ACCOUNT NUMBER]",
+                              "[CREDIT CARD NUMBER]",
+                              "[DATE OF BIRTH]",
+                              "[DATE]",
+                              "[DRIVER LICENSE NUMBER]",
+                              "[EMAIL ADDRESS]",
+                              "[RACE/ETHNICITY]",
+                              "[GENDER]",
+                              "[IP ADDRESS]",
+                              "[LOCATION]",
+                              "[MONEY]",
+                              "[ORGANIZATION]",
+                              "[PASSPORT NUMBER]",
+                              "[PASSWORD]",
+                              "[PERSON NAME]",
+                              "[PHONE NUMBER]",
+                              "[PROFESSION]",
+                              "[SOCIAL SECURITY NUMBER]",
+                              "[USERNAME]",
+                              "[SECRET_KEY]",
+                              "[TRANSACTION_ID]",
+                              "[RELIGION]",
+                          ],
+                          "contextual_rules":[
+                                  ["[PERSON NAME]", "[EMAIL ADDRESS]", "[LOCATION]" ],
+                                  [ "[PERSON NAME]", "[CREDIT CARD NUMBER]", "[BANK ACCOUNT NUMBER]" ],
+                                  [ "[PERSON NAME]", "[EMAIL ADDRESS]", "[DATE OF BIRTH]" ]
+                          ],
+                          "matching_scores": {
+                              "[BANK ACCOUNT NUMBER]": 0.5,
+                              "[CREDIT CARD NUMBER]": 0.5,
+                              "[DATE OF BIRTH]": 0.5,
+                              "[DATE]": 0.5,
+                              "[DRIVER LICENSE NUMBER]": 0.5,
+                              "[EMAIL ADDRESS]": 0.5,
+                              "[RACE/ETHNICITY]": 0.5,
+                              "[GENDER]": 0.5,
+                              "[IP ADDRESS]": 0.5,
+                              "[LOCATION]": 0.5,
+                              "[MONEY]": 0.5,
+                              "[ORGANIZATION]": 0.5,
+                              "[PASSPORT NUMBER]": 0.5,
+                              "[PASSWORD]": 0.5,
+                              "[PERSON NAME]": 0.5,
+                              "[PHONE NUMBER]": 0.5,
+                              "[PROFESSION]": 0.5,
+                              "[SOCIAL SECURITY NUMBER]": 0.5,
+                              "[USERNAME]": 0.5,
+                              "[SECRET_KEY]": 0.5,
+                              "[TRANSACTION_ID]": 0.5,
+                              "[RELIGION]": 0.5
+                          }
+                        },
+                        "confidential_detection": {
```

Comment:
the indent looks a little off here?

Reply:
tried to resolve it in subsequent commit